extension-requests/script.js

```diff
@@ -1160,17 +1260,29 @@ async function createExtensionCard(data, dev) {
     updateExtensionRequest({
       id: data.id,
       body: formData,
+      dev: dev,
```

Comment:
what is this dev=dev ..?

Reply:
Changed the confusing variable name to `underDevFeatureFlag`